picard/mbjson.py

```diff
@@ -306,76 +306,76 @@ def check_higher_score(locale_dict, locale, score):
                 root_parts.append((score, 5))
                 comb = linear_combination_of_weights(full_parts)
                 if check_higher_score(full_locales, full_locale, comb):
-                    full_locales[full_locale] = (comb, (alias['name'], alias["sort-name"]))
+                    full_locales[full_locale] = (comb, (alias['name'], alias['sort-name']))
                 comb = linear_combination_of_weights(root_parts)
                 if check_higher_score(root_locales, root_locale, comb):
-                    root_locales[root_locale] = (comb, (alias['name'], alias["sort-name"]))
+                    root_locales[root_locale] = (comb, (alias['name'], alias['sort-name']))
 
             # First pass to match full locale if available
-            for locale in config.setting["artist_locales"]:
+            for locale in config.setting['artist_locales']:
                 if locale in full_locales:
                     return full_locales[locale][1]
 
             # Second pass to match root locale if available
-            for locale in config.setting["artist_locales"]:
-                lang = locale.split("_")[0]
+            for locale in config.setting['artist_locales']:
+                lang = locale.split('_')[0]
                 if lang in root_locales:
                     return root_locales[lang][1]
 
         # No matches found in available alias locales
-        translsort = node['sort-name']
-        transl = translate_from_sortname(node['name'] or "", translsort)
+        sort_name = node['sort-name']
+        translated_name = translate_from_sortname(node['name'] or "", sort_name)
```

Comment:
As you are at it, you could also change `""` to `''` here

Reply:
Done in [a972b72](https://github.com/metabrainz/picard/pull/2216/commits/a972b72ca69145848596f5fe3ef20852fad755c9)